lib/account/account-manager.js

```diff
@@ -1,36 +1,80 @@
 "use strict";
-/**
- * Manages the business logic of account controls.
- */
 
 /**
- * The object sent in which has information needed to validate the user
+ * The object sent container information needed to validate the user
  * has properly set up their two factor authentication and has generated
- * a password for use to save.
+ * a password for use.
  *
  * @typedef {Account~accountinfo}
- * @param {string} accountId this is the unhashed account id sent to client
- * @param {string} currentMfa
- * @param {string} previousMfa
- * @param {string} password
+ * @property {string} accountId this is the unhashed account id sent to client
+ * @property {string} currentMfa
+ * @property {string} previousMfa
+ * @property {string} password
  */
-class AccountManager {
+
+/**
+ * This is what is sent back to account holder to confirm their
+ * sign up by showing the password salt and mfa key we've
+ * generated for them.
+ *
+ * @typedef {Account~confirm}
```

Comment:
The typedef should be `accountManager~confirm` instead of `Account~confirm`.

* accountManager is not a class.
* There is no Account class.